streams/src/test/java/org/apache/kafka/streams/processor/internals/RecordCollectorTest.java

```diff
@@ -548,9 +548,9 @@ public Optional<Set<Integer>> partitions(final String topic, final String key, f
         final Map<TopicPartition, Long> offsets = collector.offsets();
 
         // with mock producer without specific partition, we would use default producer partitioner with murmur hash
```

Comment:
We use RoundRobinPartitioner instead of "default producer partitioner with murmur hash"; should we update the comment?

Reply:
I think RoundRobinPartitioner does not use murmur hash? So we should remove it from the comments.

Reply:
Yes, it doesn't use murmur hash. Removed it. Thanks.